src/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixer.php

```diff
@@ -100,10 +100,25 @@ final class SingleSpaceAfterConstructFixer extends AbstractFixer implements Conf
         'yield_from' => T_YIELD_FROM,
     ];
 
+    private SingleSpaceAroundConstructFixer $singleSpaceAroundConstructFixer;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct()
+    {
+        $this->singleSpaceAroundConstructFixer = new SingleSpaceAroundConstructFixer();
+
+        parent::__construct();
+    }
+
     /**
-     * @var array<string, int>
+     * {@inheritdoc}
```

Comment:
Side note, nothing to be done: I find such phpDocs with only `{@inheritdoc}` superfluous.

Reply:
to be consider as practice to use or not to use them for whole project. probably a fixer, if we want to remove them.